_sass/page.scss

```diff
@@ -29,7 +29,8 @@ body.dark {
 .banner {
   padding: 20px 0px 50px 0px;
   height: 150px;
-  background-color: #1d539f;
+  // background-color: #133667;
```

Comment:
I know you're still working on this PR but the final draft shouldn't include any commented out code.